guess_number/guess_word.py

```diff
@@ -0,0 +1,25 @@
+"""Pick a random word and have the user try to guess it. Provide feedback on the guessed word"""
+import random
+
+words = ["a", "be", "cat", "dog", "elf"]
+
+random_word = random.choice(words)
+
+print("Here are the list of words: ")
+for word in words:
+    print(word)
+
+print("\n")
+
+
+while True:
+    user_guess = input("Guess a word from the list of words: ")
+    if user_guess == random_word:
+        print("You guessed it!")
+        break
+
+    elif random_word[0] > user_guess[0]:
```

Comment:
I _believe_ you can just use `random_word > user_guess` because the > should base the comparison on the alphabetized order.